checks/raw/fuzzing.go

```diff
@@ -322,7 +377,7 @@ func getProminentLanguages(langs []clients.Language) []clients.LanguageName {
 	// Languages that have lines of code above average will be considered prominent.
 	ret := []clients.LanguageName{}
 	for lName, loC := range langMap {
-		if loC >= avgLoC {
+		if loC >= avgLoC/4.0 {
```

Comment:
where is this magic 4.0 coming from?

Reply:
Minor empirical testing. I ran it through a testing repo where I was confused why some fuzzers present wasn't showing up. The cause was the project had 2 languages and due to the language with fewer lines of code being the one with the fuzzer it wasn't showing up. Then, I didn't see a strong argument for why we need the average and would like to lower it significantly. Examples where the average is relevant is e.g. https://github.com/Instagram/LibCST where the native code in Rust has ~14% of the coverage, and Python has the rest. What was the reason for having the average mark?

Reply:
Written before my time, but my guess is we were concerned about if the main language was fuzzed.

If the Rust code is fuzzed, but the Python isn't does that project deserve a 10?

Reply:
If keeping the 4.0, can you make it a constant with a constant expressing this?

Reply:
> Written before my time, but my guess is we were concerned about if the main language was fuzzed.
> If the Rust code is fuzzed, but the Python isn't does that project deserve a 10?

I think in the current spirit of the fuzzing checks then the answer is yes it deserves a 10. However, I think medium-term it makes sense to consider a more refined fuzzing assessment, including some of the points considered in https://github.com/ossf/scorecard/issues/3475

Reply:
Sorry to belabor this, but naming the constant `fourPointZero` doesn't help with readability compared to just using 4.0, which is what my initial comment was about.

I was thinking something more similar to:
```go
	// Calculate the average lines of code in the current repo.
	// This var can stay as an int, no need for a precise float value.
	avgLoC := totalLoC / numLangs
	// Languages that have lines of code above 1/8th will be considered prominent.
	prominentThreshold = avgLoC / 4.0
	ret := []clients.LanguageName{}
	for lName, loC := range langMap {
		if loC >= prominentThreshold {
```

Reply:
Set it to what you pasted